test-app/src/main/java/net/rubygrapefruit/platform/test/Main.java

```diff
@@ -19,21 +19,40 @@
 import joptsimple.OptionException;
 import joptsimple.OptionParser;
 import joptsimple.OptionSet;
-import net.rubygrapefruit.platform.*;
+import net.rubygrapefruit.platform.Native;
+import net.rubygrapefruit.platform.NativeIntegrationUnavailableException;
 import net.rubygrapefruit.platform.Process;
-import net.rubygrapefruit.platform.file.*;
+import net.rubygrapefruit.platform.SystemInfo;
+import net.rubygrapefruit.platform.file.DirEntry;
+import net.rubygrapefruit.platform.file.FileInfo;
+import net.rubygrapefruit.platform.file.FileSystemInfo;
+import net.rubygrapefruit.platform.file.FileSystems;
+import net.rubygrapefruit.platform.file.FileWatcher;
+import net.rubygrapefruit.platform.file.FileWatcherCallback;
+import net.rubygrapefruit.platform.file.Files;
+import net.rubygrapefruit.platform.file.PosixFileInfo;
+import net.rubygrapefruit.platform.file.PosixFiles;
+import net.rubygrapefruit.platform.internal.Platform;
+import net.rubygrapefruit.platform.internal.jni.OsxFileEventFunctions;
+import net.rubygrapefruit.platform.internal.jni.WindowsFileEventFunctions;
```

Comment:
I don't think we should add this until there is a public API for file events, as this is also a sample and documentation for the APIs.

Reply:
So you think we should remove it from `master` again for the time being? It was very helpful to use the test-app to debug problems with file watching.

Reply:
That's one option. The other is to add a public API, which is what I would do. We just need to extract an interface or two out of the implementations.